Add english docs for UIE (#4255)
PaddlePaddle/PaddleNLP: applications/information_extraction/taskflow_text_en.md

# UIE Taskflow User Guide - Text Information Extraction

**Table of contents**

> - [1. Function Introduction](#1)
> - [2. Application Example](#2)
> - [3. Text Information Extraction](#3)
>    - [3.1 Entity Extraction](#31)
>    - [3.2 Relation Extraction](#32)
>    - [3.3 Event Extraction](#33)
>    - [3.4 Opinion Extraction](#34)
>    - [3.5 Sentiment Classification](#35)
>    - [3.6 Cross-task Extraction](#36)
>    - [3.7 Model Selection](#37)
>    - [3.8 More Configuration](#38)

Reviewer: ```suggestion
- [1. Introduction](#1)
- [2. Examples](#2)
- [3. Text Information Extraction](#3)
   - [3.1 Entity Extraction](#31)
   - [3.2 Relation Extraction](#32)
   - [3.3 Event Extraction](#33)
   - [3.4 Opinion Extraction](#34)
   - [3.5 Sentiment Classification](#35)
   - [3.6 Multi-task Extraction](#36)
   - [3.7 Available Models](#37)
   - [3.8 More Configuration](#38)
```

<a name="1"></a>

> ## 1. Function introduction
> 
> ```paddlenlp.Taskflow``` provides plain text universal information extraction, opinion extraction and other capabilities, and can extract various types of information, including but not limited to named entity recognition (such as person name, place name, organization name, etc.), relationship (such as the director of the movie, the release time of the song, etc.), events (such as a car accident at a certain intersection, an earthquake in a certain place, etc.), and information such as evaluation dimensions, opinion words, and emotional tendencies. Users can use natural language to customize the extraction target, and can uniformly extract the corresponding information in the input text without training. ** Realize out-of-the-box use and meet various information extraction needs **

Reviewer: ```suggestion
## 1. Introduction
```paddlenlp.Taskflow``` provides general information extraction of text and documents, evaluation opinion extraction and other capabilities, and can extract various types of information, including but not limited to named entities (such as person name, place name, organization name, etc.), relations (such as the director of the movie, the release time of the song, etc.), events (such as a car accident at a certain intersection, an earthquake in a certain place, etc.), and information such as product reviews, opinions, and sentiments. Users can use natural language to customize the extraction target, and can uniformly extract the corresponding information in the input text or document without training.
```

<a name="2"></a>

> ## 2. Application examples

Reviewer: ```suggestion
## 2. Examples
```

> ## 2. Application examples

Reviewer: ```suggestion
## 2. Examples
```

                                             'probability': 0.9072940447883724,
                                             'start': 15,
                                             'text': 'very nice'}],
                                 'Sentiment classification [negative, positive]': [{'probability': 0.9998571920670685,
                                                                                   'text': 'positive'}]},
                   'start': 4,
                   'text': 'teacher'}]}]
     ```

<a name="35"></a>

#### 3.5 Sentiment Classification

   - Sentence-level sentiment classification, that is, to judge whether the emotional orientation of a sentence is "positive" or "negative". The schema structure is as follows:

     ```text
     'Sentiment classification [negative, positive]'
     ```

     Example:

     ```python
     >>> schema = 'Sentiment classification [negative, positive]'
     >>> ie_en.set_schema(schema)
     >>> ie_en('I am sorry but this is the worst film I have ever seen in my life.')
     [{'Sentiment classification [negative, positive]': [{'text': 'negative', 'probability': 0.9998415771287057}]}]
     ```

> #### 3.6 Cross-task extraction

Reviewer: ```suggestion
#### 3.6 Multi-Task Extraction
```

      }
    ]
    ```

    Example:

    ```python
    >>> schema = ['法院', {'原告': '委托代理人'}, {'被告': '委托代理人'}]
    >>> ie.set_schema(schema)
    >>> pprint(ie("北京市海淀区人民法院\n民事判决书\n(199x)建初字第xxx号\n原告：张三。\n委托代理人李四，北京市 A律师事务所律师。\n被告：B公司，法定代表人王五，开发公司总经理。\n委托代理人赵六，北京市 C律师事务所律师。")) # Better print results using pprint
    [{'原告': [{'end': 37,
              'probability': 0.9949814024296764,
              'relations': {'委托代理人': [{'end': 46,
                                      'probability': 0.7956844697990384,
                                      'start': 44,
                                      'text': '李四'}]},
              'start': 35,
              'text': '张三'}],
      '法院': [{'end': 10,
              'probability': 0.9221074192336651,
              'start': 0,
              'text': '北京市海淀区人民法院'}],
      '被告': [{'end': 67,
              'probability': 0.8437349536631089,
              'relations': {'委托代理人': [{'end': 92,
                                      'probability': 0.7267121388225029,
                                      'start': 90,
                                      'text': '赵六'}]},
              'start': 64,
              'text': 'B公司'}]}]
    ```

<a name="37"></a>

> #### 3.7 Model Selection
> 
> - Multiple model options to meet accuracy and speed requirements

Reviewer: ```suggestion
#### 3.7 Available Model

- A variety of models to different accuracy and speed requirements
```

Show less
1,062 / 5,000
Translation results

> Translation result

Reviewer: 这是啥？看看 @linjieccc

Author: 已删除
